resources/levels-progression/application.json

```diff
@@ -27,7 +27,7 @@
         {
             "board": ["_(5)", "_(3)", "_(8)"],
             "goal": ["1", "6", "8"],
-            "toolbox": ["x => x + 1", "x => x - 2", "x => x", "x => x + 3"]
+            "toolbox": ["x => x + 1", "x => x + 1", "x => x - 2", "x => x", "x => x + 3"]
```

Comment:
Shouldn't this be an apply block, not just another copy of a function? Also, I think this is the wrong level - level 71 has two `__unlimited` things in its toolbox, not 4 functions.